lib/cli/commands/listpairs.ts

```diff
@@ -1,11 +1,38 @@
 import { callback, loadXudClient } from '../command';
 import { Arguments } from 'yargs';
-import { ListPairsRequest } from '../../proto/xudrpc_pb';
+import { ListPairsRequest, ListPairsResponse } from '../../proto/xudrpc_pb';
+import Table, { HorizontalTable } from 'cli-table3';
+import colors from 'colors/safe';
+import { getPairId } from '../../utils/utils';
+
+const HEADERS = [
+  colors.red('PairId'),
+  colors.green('Base'),
+  colors.red('Quote'),
+];
+
+const formatPairs = (pairs: ListPairsResponse.AsObject): string[][] => {
+  const formatted: string[][] = [];
+  pairs.pairsList.forEach((pair) => {
+    const pairString = getPairId(pair);
```

Comment:
This variable isn't actually a string, it's an array, so the naming is misleading here. But again I think you can just use `const [baseCurrency, quoteCurrency] = pairId.split('/');` here and then use those two variables in the line below.